sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -345,7 +358,8 @@ object ScalaNativePluginInternal {
         }
         val libunwind = os match {
           case "Mac OS X" => Seq.empty
-          case _          => Seq("unwind", "unwind-" + arch)
+          case _ =>
+            if (isWindows) Seq.empty else Seq("unwind", "unwind-" + arch)
```

Comment:
Adding a `case "Windows"=> Seq.empty` or using the special case for macOS would be more in the spirit of the existing code.